### ISSUE TITLE
Accessible progress bar with multiple values

### ISSUE THREAD

**Issue by waterplea** [w3c/aria-practices#1791]
Hi all. I couldn't find any info on the subject. What if I want to create a progress bar indicator with multiple values? Kind of like a video progress with watched and buffered regions. Is there any accessibility guidelines for such a use case?

**Comment by JAWS-test** [comment]
A progressbar can only have one value (aria-valuenow). That should be the more important value (watched time). The second value can be transmitted in the description (aria-describedby) or both values in the aria-valuetext.

Usually, the time scrubber can not only display the time, but also change it. Then progressbar would be wrong and you should use a slider instead. There are [multi thumb sliders](https://www.w3.org/TR/wai-aria-practices-1.2/examples/slider/multithumb-slider.html), but that would not be suitable for a video, as only one value can be changed (the watched time and not the buffered regions). Possibly use a slider for watched time and a progressbar for buffered regions.

There does not seem to be an optimal solution at the moment.

**Comment by waterplea** [comment]
Thanks a lot for the explanation! I think I can work something out with your tips.
